docs/about/maintainers.rst

```diff
@@ -0,0 +1,32 @@
+.. raw :: html
+
+    <!-- Generated by generate_maintainers_table.py -->
+    <div class="maintainers-container">
+    <style>
+      img.avatar {border-radius: 10px;}
+    </style>
+    <div>
+    <a href='https://github.com/adrinjalali'><img src='https://github.com/adrinjalali.png' class='avatar' /></a> <br />
```

Comment:
Is this what's generated by the script below?

Reply:
See PR description. Without that file you can't run the basic doc build (without multiversion). Also, I don't expect it to change every other week.

Reply:
I'm also a bit confused as why the single doc build would fail w/o this file. We need to fix our whole doc build :see_no_evil:

Reply:
Well tbh I didn't try but if you have a missing file it should at least complain, shouldn't it? In any case it's rerun during multiversion build.

I consider the single version doc build a feature rather than a bug. I learned a lot about scikit-learn's doc build the past week and see how it works better in some ways, but it can't help you switch between versions of the same page (only to the start page of another version).

Reply:
Correction: it doesn't fail, just prints the following and doesn't add anything.
```
fairlearn\docs\about\index.rst:152: WARNING: Problems with "include" directive path:
InputError: [Errno 2] No such file or directory: 'docs/about/maintainers.rst'.
```
![image](https://user-images.githubusercontent.com/10245648/126269290-d485955b-12ff-4039-ad7a-1ff160ab86ac.png)

Reply:
I removed it in the latest commit.